.github/workflows/codecov_ats.yml

```diff
@@ -132,8 +132,7 @@ jobs:
               "--ignore=tests/sentry/post_process_forwarder",
               "--ignore=tests/sentry/snuba",
               "--ignore=tests/sentry/search/events",
-              "--ignore=tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py",
-              "--ignore=tests/sentry/region_to_control/test_region_to_control_kafka.py"
```

Comment:
Hmm what prompted this change?

Reply:
Just saw a reference to a file that no longer exists.